docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Considerations for Pod Resource (Memory and CPU) Requests and Limits"
+date: 2020-06-30T08:55:00-05:00
+draft: true
+weight: 40
+---
+The operator creates a pod for each running WebLogic Server instance and each pod will have a container. It.s important that containers have enough resources in order for applications to run efficiently and expeditiously. 
+
+If a pod is scheduled on a node with limited resources, it.s possible for the node to run out of memory or CPU resources, and for applications to stop working properly or have degraded performance. It.s also possible for a rouge application to use all available memory and/or CPU, which makes other containers running on the same system unresponsive. The same problem can happen if an application has memory leak or bad configuration. 
```

Comment:
rouge -> rogue (took me a bit to figure that one out - interesting autocorrect :-) )